internal/monitor/instance/linux/processor.go

```diff
@@ -293,7 +319,7 @@ func (l *linuxProcessor) ReSync(e *events.EventInfo) error {
 		reacquired = append(reacquired, eventInfo.PUID)
 
 		// Synchronize
-		if l.config.SyncHandler != nil {
+		if l.config.SyncHandler != nil && storedContext.Tags != nil && len(storedContext.Tags.GetSlice()) != 0 {
```

Comment:
Will storedContext.Tags ever be nil given that we have remap ? Maybe we should have a check that once we retrieve the tags are not nil and not have the rest of the code keep checking this.